internal/npm_install/node_module_library.bzl

```diff
@@ -27,19 +27,35 @@ def _node_module_library_impl(ctx):
             scripts = depset(transitive = [scripts, src[NodeModuleSources].scripts])
     scripts = depset(ctx.files.scripts, transitive = [scripts])
 
-    return [
-        DefaultInfo(
-            files = sources,
-        ),
-        NodeModuleInfo(
-            workspace = workspace,
-        ),
-        NodeModuleSources(
-            sources = sources,
-            scripts = scripts,
-            workspace = workspace,
+    declarations = [f for f in ctx.files.srcs if f.path.endswith("d.ts")]
```

Comment:
this matches a bit too much like `pond.ts`

Reply:
🤦‍♂ good catch. thanks.